### PR TITLE
adding more blogs

### PR DESCRIPTION
**Pull request by Heulitig**

### CODE REVIEW

File: blog/loops.ftd [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+-- common.post-meta meta: Understanding Loops
+published-on: November 8, 2023 at 3:29 pm
+post-url: /loops/
+author: $authors.rithik
+read-time: 5-10 mins
+
+In this blog, we will be exploring on how `fastn` uses loops and
+what cool stuff you can do with them. Let's kick things
+off by using some lists. We'll explore and learn about
+different interesting aspects while working with them.
```
Comment (Arpita-Jaiswal) [inline blog/loops.ftd:29]: Can we combine `index-meta` and `meta` variables into one.

File: blog/loops.ftd [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+- [Using for loop syntax](/loops#looping-using-for-loop-syntax)
+- [Using $loop$ syntax](/loops#looping-using-loop-syntax)
+
+-- ds.h2: Looping using for loop syntax
+
+Using for syntax is recommended for looping and has the following
+general syntax that we should keep in mind before using it.
+
+`for: <LIST-ITEM-NAME>, [<LOOP-INDEX>] in <LIST-NAME>`
```
Comment (Arpita-Jaiswal) [inline blog/loops.ftd:79]: [<LOOP-INDEX>] doesn't work if imported in other module. When should I expect the fix would come? I hope before the release of this blog post. 
@Heulitig

File: blog/loops.ftd (hunk 1 above)
Comment (Heulitig) [reply blog/loops.ftd:29]: I will do that

File: blog/loops.ftd (hunk 2 above)
Comment (Heulitig) [reply blog/loops.ftd:79]: I will fix that before this post

File: FASTN.ftd [hunk 3]
```diff
@@ -933,7 +933,8 @@ skip: true
   document: blog/external-css.ftd
 - Using Custom Breakpoints: /breakpoint/
   document: blog/breakpoint.ftd
- 
+- Understanding Loops: /loops/
+  document: blog/loops.ftd
```
Comment (amitu) [inline FASTN.ftd:937]: Change the url to `/loop/`, so it matches with our `/record/`, `/list/` etc URL. Else we will have to think if it is `/loop/` or `/loops/`. Also add a redirect entry from `/loops/ -> /loop/`.

**Comment by amitu** [comment]
Regarding loop content, please ensure `/list/` page also has all relevant information, or at least a link to `/loops/` because once someone learns about list they have to learn about loops.

File: ftd/loop.ftd [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+
+
+
+
+
+
+
+
+-- ds.h2: Looping using the `$loop$` syntax
```
Comment (Arpita-Jaiswal) [inline ftd/loop.ftd:134]:
```suggestion
-- ds.h2: Looping using the `$loop$` syntax (Deprecated Syntax)
```

File: blog/loops.ftd (hunk 2 above)
Comment (Heulitig) [reply blog/loops.ftd]: This is fixed. Conditional loops still has some issue inside ftd.ui lists which I will fix later and update its content in this doc after those fixes.

File: cs/create-cs.ftd [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.9.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.h1: Configure index.ftd
+
+Before we start using our colors, we need to configure some details
+defined inside `index.ftd`.  Initially, the `index.ftd` will look like this.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.10.png
```
Comment (amitu) [inline cs/create-cs.ftd:112]: Let's use `ds.code` to show code snippets.

File: cs/create-cs.ftd [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+
+-- ds.markdown:
+
+After modifying index.ftd, we will see the contents like this.
+
+-- ftd.image: $fastn-assets.files.images.cs.how-to-create-cs.13.png
+height.fixed.px: 400
+margin-vertical.px: 10
+
+-- ds.h1: Let's make our repository live
```
Comment (amitu) [inline cs/create-cs.ftd:144]: There is a lot of copy paste of this material. Write it in a component and use it from everywhere.

**Review by Arpita-Jaiswal — APPROVED** [review]

**Pull request merged**